doc/source/science_guide/sg_dynamics.rst

```diff
@@ -236,8 +236,32 @@ pending further testing.
 Seabed stress
 ***************
 
-The parameterization for the seabed stress is described in :cite:`Lemieux16`. The components of the basal seabed stress are 
-:math:`\tau_{bx}=C_bu` and :math:`\tau_{by}=C_bv`, where :math:`C_b` is a coefficient expressed as
+CICE includes two options for calculating the seabed stress,
+i.e. the term in the momentum equation that represents the interaction
+between grounded ice keels and the seabed. The seabed stress can be
+activated by setting ``seabed_stress`` to true in the namelist. The seabed stress (or basal
+stress) parameterization of :cite:`Lemieux16` is chosen if ``seabed_stress_method``
+= ``LKD`` while the new probabilistic approach is used if ``seabed_stress_method``
+= ``probabilistic``. 
+
+For both parameterizations, the components of the seabed
+stress are expressed as :math:`\tau_{bx}=C_bu` and
+:math:`\tau_{by}=C_bv`, where :math:`C_b` is a seabed stress
+coefficient.
+
+The two parameterizations differ in their calculation of
+the :math:`C_b` coefficients. 
+
+Note that the user must provide a bathymetry field for using these
+grounding schemes. It is suggested to have a bathymetry field with water depths
+larger than 5 m that represents well shallow water (less than 30 m) regions such as the Laptev Sea
+and the East Siberian Sea.   
+
+Seabed stress based on linear keel draft
```

Comment:
Add (LKD) to the end of this line - I don't see it defined elsewhere in the documentation (but maybe it's there and just not showing up on the github code comparison)